tests/e2e-utils/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@woocommerce/e2e-utils",
+    "version": "1.0.0",
```

Comment:
Can we start with version `0.1.0` so that we have room to test and make adjustments prior to releasing `1.0.0` ?